tests/files/c_source/vehicle.h

```diff
@@ -256,6 +256,56 @@
 #define VEHICLE_FRAME_ID_RT_SB_ACCEL (0x94a6c22U)
 #define VEHICLE_FRAME_ID_RT_SB_GYRO_RATES (0x94a6d22U)
 
+#define VEHICLE_RT_SB_INS_STATUS_INS_STATUS_VAL_CONVERGED (2U)
+#define VEHICLE_RT_SB_INS_STATUS_INS_STATUS_VAL_INITIALISED (1U)
+#define VEHICLE_RT_SB_INS_STATUS_INS_STATUS_VAL_UNINITIALISED (0U)
```

Comment:
A choice value may be negative, and even a 64 bits value. We should generate the suffix (`U` above) based on the signal type.

Reply:
I'm not sure what you mean here. Currently this should only add `U` if the signal is unsigned (`signal.is_signed`). Wouldn't this work correctly?

Or are you saying that you want to cast against the specific type depending on the length of the signal?

Reply:
Ohh, sorry, I didn't see that the `U` was only added for unsigned signals.

Yeah, I was thinking of a specific suffix depending on the type length. I know it's unlikely a choice value would not fit in an integer, but it could happen. I'm fine with adding support for this later.